### ISSUE TITLE
CVE [High] Present in Docker Image

### ISSUE THREAD

**Issue by kennytrytek-wf** [grafana/agent#6407]
### What's wrong?

```
grype --only-fixed grafana/agent:v0.39.2

 ✔ Vulnerability DB                [no update available]  
 ✔ Loaded image                                                                   grafana/agent:v0.39.2
 ✔ Parsed image                    sha256:94ef43018ee19daeb49db8b20fa36823c419f0ad32127547e15d0ebc4f913  
 ✔ Cataloged contents                  57314c18f8dc8219953ff1442c4a5811decb0b4ceef2237665c9352c05bf5990
   ├── ✔ Packages                        [685 packages]  
   ├── ✔ File digests                    [2,664 files]  
   ├── ✔ File metadata                   [2,664 locations]  
   └── ✔ Executables                     [723 executables]  
 ✔ Scanned for vulnerabilities     [38 vulnerability matches]  
   ├── by severity: 0 critical, 3 high, 16 medium, 17 low, 2 negligible
   └── by status:   26 fixed, 12 not-fixed, 0 ignored 

NAME                            INSTALLED                             FIXED-IN                 TYPE       VULNERABILITY        SEVERITY 
github.com/cloudflare/circl     v1.3.3                                1.3.7                    go-module  GHSA-9763-4f94-gfch  High      
github.com/opencontainers/runc  v1.1.9                                1.1.12                   go-module  GHSA-xr7r-f8xq-vfvv  High      
```

### Steps to reproduce

`grype --only-fixed grafana/agent:v0.39.2`

### Software version

Grafana Agent v0.39.2

**Comment by hainenber** [comment]
This should be addressed in the next minor release. Consider it done! :D
- https://github.com/grafana/agent/pull/6066
- https://github.com/grafana/agent/pull/6285

**Comment by hainenber** [comment]
Yeah, I can confirm that the affected version has been patched in `v0.40`. If possible, please retry and consider the issue closed :D
-  `circl`: https://github.com/grafana/agent/blob/release-v0.40/go.mod#L642
- `runc`: https://github.com/grafana/agent/blob/release-v0.40/go.mod#L503

**Comment by kennytrytek-wf** [comment]
```
$ grype --only-fixed grafana/agent:v0.40.0-rc.2
 ✔ Vulnerability DB                [no update available]  
 ✔ Parsed image                    sha256:65756c48306030974f1fec05072d7f71622d81c632a69f5ab90b1318edc0c  
 ✔ Cataloged contents                  7cbfb1dd2c75451d481c88e913a127c46eb24813214718ec86ae25da90157bdd
   ├── ✔ Packages                        [695 packages]  
   ├── ✔ File digests                    [2,664 files]  
   ├── ✔ File metadata                   [2,664 locations]  
   └── ✔ Executables                     [723 executables]  
 ✔ Scanned for vulnerabilities     [21 vulnerability matches]  
   ├── by severity: 0 critical, 1 high, 9 medium, 9 low, 2 negligible
   └── by status:   9 fixed, 12 not-fixed, 0 ignored 
NAME                     INSTALLED                             FIXED-IN                 TYPE       VULNERABILITY        SEVERITY 
github.com/grafana/loki  v1.6.2-0.20231004111112-07cbef92268a  2.3.0                    go-module  GHSA-grj5-8x6q-hc9q  Medium    
libc-bin                 2.38-1ubuntu6                         2.38-1ubuntu6.1          deb        CVE-2023-6780        Medium    
libc-bin                 2.38-1ubuntu6                         2.38-1ubuntu6.1          deb        CVE-2023-6779        Medium    
libc-bin                 2.38-1ubuntu6                         2.38-1ubuntu6.1          deb        CVE-2023-6246        Medium    
libc6                    2.38-1ubuntu6                         2.38-1ubuntu6.1          deb        CVE-2023-6780        Medium    
libc6                    2.38-1ubuntu6                         2.38-1ubuntu6.1          deb        CVE-2023-6779        Medium    
libc6                    2.38-1ubuntu6                         2.38-1ubuntu6.1          deb        CVE-2023-6246        Medium    
login                    1:4.13+dfsg1-1ubuntu1                 1:4.13+dfsg1-1ubuntu1.1  deb        CVE-2023-4641        Low       
passwd                   1:4.13+dfsg1-1ubuntu1                 1:4.13+dfsg1-1ubuntu1.1  deb        CVE-2023-4641        Low
```
